tests/onnx2pytorch/operations/test_expand.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import pytest
+
+from onnx2pytorch.operations.expand import Expand
+
+def test_expand_dim_changed():
+    op = Expand()
+    inp = torch.reshape(torch.arange(0, 3, dtype=torch.float32), [3, 1])
+    new_shape = [2, 1, 6]
+    exp = inp * torch.ones(new_shape)
+    exp_shape = (2, 3, 6)
+    assert tuple(op(inp, new_shape).shape) == exp_shape
```

Comment:
Here and in the test below you are calling `op(inp, new_shape)` twice. Could you rather put it in a variable and re-use it? If you don't have time it's ok, I'll just merge it.